haskell/actions.bzl

```diff
@@ -379,13 +379,16 @@ def compilation_defaults(ctx):
 
   # Common flags
   args.add([
-    "-no-link",
+    "-c",
```

Comment:
`-c` makes it stop after producing object files, this is what we probably wanted with `-no-link`. @Fuuzetsu says that `-no-link` is probably useless. Cabal passes `-c`, so I decided to pass it too, this may make building a bit faster, but I honestly have no idea if that's the case...